### ISSUE TITLE
explain_prediction is confusing for binary classifiers

### ISSUE THREAD

**Issue by kmike** [TeamHG-Memex/eli5#223]
Currently we're always showing contributions for positive class, not for predicted class:

![screen shot 2017-06-23 at 1 48 20 am](https://user-images.githubusercontent.com/107893/27455263-2a46866c-57b6-11e7-9697-d51a15b0782e.png)

Maybe it makes sense to show y=0 for the above example, with positive BIAS and score.

**Comment by lopuhin** [comment]
It's even more confusing without ``proba``... This makes sense, I agree! 👍
